plugins/vendor/sso/lib/api/KalturaSsoFilter.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * @package plugins.sso
+ * @subpackage api.filters
+ */
+class KalturaSsoFilter extends KalturaRelatedFilter

```

Comment:
Use the filter generator to generate the base class filter once that is done you can extend it and add the getListResponse function (api_v3/generator/generate_filters.php)